src/Symfony/Bundle/FrameworkBundle/Command/CachePoolDeleteCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * Delete an item from a cachhe pool.
+ *
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+final class CachePoolDeleteCommand extends Command
```

Comment:
Since this can only delete an item as specified in descriptions, wouldn't `CachePooltemDeleteCommand` be more accurate? Also, there's a tyop in the comment above.

Reply:
This matches the name of the command, and is good enough IMHO as nothing else than an item can be deleted.

Reply:
>  as nothing else than an item can be deleted.

Not everyone may know that though and have different expectations. But as long as it's documented well, it's ok.